vde_metadynamics/render_network.py

```diff
@@ -41,13 +41,16 @@ def render_print_val(arg,stride=1,file="CV"):
 def render_fc_layer(layer_indx, lp):
     output=[]
     for i in np.arange(lp.out_features):
-        if layer_indx==0:
+        if layer_indx==1:
             arg=','.join(["f%d_%d"%(layer_indx-1,j) for j in range(lp.in_features)])
         else:
             arg=','.join(["l%d_%d"%(layer_indx-1,j) for j in range(lp.in_features)])
 
         weights = ','.join(map(str,lp.weight[i].data.tolist()))
-        bias =','.join(map(str,lp.bias[i].data.tolist()))
+        try:
```

Comment:
I am guessing you have 1 feature in the input so this was causing an issue?

Reply:
I encountered this on the 2-alanine example, so there's 4 (2 sin and 2 cos).

Reply:
hmm, weird, i will investigate this further then.

Reply:
This code gives the correct behavior without in both instances as but it is possible that pytorch has changed how it returns the bias when tolist() is called. 
```python
In [12]: lp=[1]

In [13]: ','.join(map(str,lp))
Out[13]: '1'

In [14]: lp=[1,2,3,4]

In [15]: ','.join(map(str,lp))
Out[15]: '1,2,3,4'

```

Reply:
basically, we only need a str that can be streamed to text file for plumed

Reply:
Yes, the problem I was running in to was that `tolist` was returning a scalar, i.e .`','.join(map(str, 1))`